### ISSUE TITLE
data cache problems

### ISSUE THREAD

**Issue by jinxiao** [marcusolsson/grafana-json-datasource#19]
when i use a json api datasource , i found a data cache problem.
i tried to update my json api , but the result on grafana not changed .
when i use curl command , it could show latest result , but grafana cannot.

**Comment by marcusolsson** [comment]
What have you set the Cache time to? By default it caches the response for 5 minutes. If you want to disable caching, you can set it to 0 seconds.

<img width="217" alt="Screen Shot 2020-12-01 at 07 19 24" src="https://user-images.githubusercontent.com/8396880/100704292-8c378000-33a5-11eb-83e1-0db0df89a364.png">

**Comment by jinxiao** [comment]
well , i have already tried it, it not always worked .
let me take an example ,
i change the datasource url, and set the cache time to zero, what showed in query inspector is still the old datasource

**Comment by marcusolsson** [comment]
Good point. We should probably invalidate the cache if the data source configuration has changed, which I don't believe we do today.
